content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md

```diff
@@ -121,22 +121,22 @@ To troubleshoot, make sure the YAML files in your __data_ folder follow these ru
 
 {% data reusables.pages.yaml-linter %}
 
-For more information about Jekyll data files, see [Data Files](https://jekyllrb.com/docs/datafiles/) in the Jekyll documentation.
```

Comment:
@akordowski Similar situation here! It looks like these quotation marks were recently removed by our team in an update - https://github.com/github/docs/commit/25182f431f6d533fd41ce89c4320a7b530d11a22

Can we ensure the quotation marks remain absent? 💛